script.py

```diff
@@ -38,16 +38,16 @@
                 command = input("Enter instruction: ")
                 continue
 
-            if len(indices) == 0:
+            if len(indices) == 0 or len(instr) == 1:
```

Comment:
Every instruction will make the models None making, thus training or viewing all models. This must be removed I guess.